synthesis_template/CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+#******************************************************************************
+# Copyright (c) 2020, Intel Corporation. All rights reserved.
+# 
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception.
+# 
+# *****************************************************************************
+
+cmake_minimum_required(VERSION 3.12)
+
+enable_testing()
+
+if(NOT DEFINED ENV{ICSC_HOME})
```

Comment:
Durch das wird es nicht gebaut oder? Dann könnte man es auch in das globale CMakeLists hinzufügen, oder?

Reply:
Ne, da bricht meine ganze Makefile ab, man hat also keins der neuen targets. Es müsste nur irgendwie die SystemC Dependency finden, das krieg ich bei mir nicht hin